"ci/kokoro/docker/build.sh cmake-super" hangs

```
$ ci/kokoro/docker/build.sh cmake-super
...
21/28 Test #21: optional_test .....................   Passed    0.01 sec
      Start 22: status_or_test
22/28 Test #22: status_or_test ....................   Passed    0.01 sec
      Start 23: status_test
23/28 Test #23: status_test .......................   Passed    0.00 sec
      Start 24: terminate_handler_test
24/28 Test #24: terminate_handler_test ............   Passed    0.02 sec
      Start 25: assert_ok_test
25/28 Test #25: assert_ok_test ....................   Passed    0.01 sec
      Start 26: completion_queue_test
```
then, if you interrupt it after waiting ...
```
^CCMakeFiles/google-cloud-cpp-project.dir/build.make:119: recipe for target 'build/google-cloud-cpp/src/google-cloud-cpp-project-stamp/google-cloud-cpp-project-test' failed
make[2]: *** [build/google-cloud-cpp/src/google-cloud-cpp-project-stamp/google-cloud-cpp-project-test] Interrupt
CMakeFiles/Makefile2:103: recipe for target 'CMakeFiles/google-cloud-cpp-project.dir/all' failed
make[1]: *** [CMakeFiles/google-cloud-cpp-project.dir/all] Interrupt
Makefile:83: recipe for target 'all' failed
make: *** [all] Interrupt
Build finished with 130 exit status Fri Jan 31 12:54:29 EST 2020.
```

--- a/bazel/google_cloud_cpp_common_deps.bzl
+++ b/bazel/google_cloud_cpp_common_deps.bzl
@@ -65,12 +65,11 @@ def google_cloud_cpp_common_deps():
     if "com_github_grpc_grpc" not in native.existing_rules():
         http_archive(
             name = "com_github_grpc_grpc",
-            strip_prefix = "grpc-1.26.0",
+            strip_prefix = "grpc-78ace4cd5dfcc1f2eced44d22d752f103f377e7b",
             urls = [
-                "https://github.com/grpc/grpc/archive/v1.26.0.tar.gz",
-                "https://mirror.bazel.build/github.com/grpc/grpc/archive/v1.26.0.tar.gz",
+                "https://github.com/grpc/grpc/archive/78ace4cd5dfcc1f2eced44d22d752f103f377e7b.tar.gz",
             ],
-            sha256 = "2fcb7f1ab160d6fd3aaade64520be3e5446fc4c6fa7ba6581afdc4e26094bd81",
+            sha256 = "a2034a1c8127e35c0cc7b86c1b5ad6d8e79a62c5e133c379b8b22a78ba370015",
         )
 
     # We use the cc_proto_library() rule from @com_google_protobuf, which

--- a/super/external/grpc.cmake
+++ b/super/external/grpc.cmake
@@ -23,9 +23,10 @@ if (NOT TARGET grpc-project)
     # Give application developers a hook to configure the version and hash
     # downloaded from GitHub.
     set(GOOGLE_CLOUD_CPP_GRPC_URL
-        "https://github.com/grpc/grpc/archive/v1.26.0.tar.gz")
+        "https://github.com/grpc/grpc/archive/78ace4cd5dfcc1f2eced44d22d752f103f377e7b.tar.gz"
+    )
     set(GOOGLE_CLOUD_CPP_GRPC_SHA256
-        "2fcb7f1ab160d6fd3aaade64520be3e5446fc4c6fa7ba6581afdc4e26094bd81")
+        "a2034a1c8127e35c0cc7b86c1b5ad6d8e79a62c5e133c379b8b22a78ba370015")
 
     set_external_project_build_parallel_level(PARALLEL)
     set_external_project_vars()

